exercises/01-getting-started.md

```diff
@@ -1,63 +1,77 @@
+Follow these [instructions](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server#how-to-run-the-server) on how to set up your SAP S/4HANA and SAP SuccessFactors mock server and deploy it on the SAP Business Technology Platform (SAP BTP), Cloud Foundry environment. 
 
-## Setup the destinations environment variable
+**Please checkout the branch [`mock-server-update-data`](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server-update-data) for this tutorial instead of the branch [`mock-server`](https://github.com/SAP/cloud-s4-sdk-book/tree/mock-server) as it is described in the instructions.** The branch has the latest date data as opposed to the branch `mock-server`.
 
-Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system.
+## Setup the destinations environment variable
 
+Replace the URL placeholders for the `S4HANA` and `SFSF` destinations in the [`.env`](../.env) file with the URL(s) of your mock server or your SAP S/4HANA and SAP SuccessFactors systems. If you are using the mock server, the URLs for both destinations are the same, because the mock server acts as both, a mocked SAP S4/HANA system and a mocked SAP SuccessFactors system. The mock server has no authentication to access it by default. Please set the authentication type `NoAuthentication` when you use the mock server.
 
 > ### How to find your mock server's URL
+>
 > Open the [SAP BTP Cockpit](https://account.hana.ondemand.com) and open the subaccount to which the mockserver was deployed. Open the space, where the mock server was deployed. You should find the URL to the mock server here.
 
 ## Start the application locally
-Open the project in your IDE. For the remainder of this course we will assume you are using Visual Studio Code. Please adapt accordingly if you are using a different IDE.
 
-To open the project go to *File > Open...* and select the folder you just cloned from GitHub.
+Open the project in your IDE. For the remainder of this tutorial, we will assume you are using Visual Studio Code. Please adapt accordingly if you are using a different IDE.
 
-Let's check that everything works and run the application locally. First, open the command line with *Terminal > New Terminal*. Then, install the dependencies:
+To open the project go to _File > Open..._ and select the folder you just cloned from GitHub.
+
+Let's check that everything works and run the application locally. First, open the command line with _Terminal > New Terminal_. Then, install the dependencies:
 
 ```sh
 npm install
 ```
 
-Deploy a local database (locally we use sqlite):
+Deploy a local database (locally we use SQLite):
+
+```sh
+npm run cds-deploy
+```
+
+Start the local database:
```

Comment:
[pp] Perhaps explain that this steps add the data from the `.csv` files to the local DB connected to the CAP service.

Reply:
I added additional explanations of the commands